src/views/dialogs/GeoComplianceDialog.tsx

```diff
@@ -15,18 +17,27 @@ import { Dialog, DialogPlacement } from '@/components/Dialog';
 import { FormInput } from '@/components/FormInput';
 import { SearchSelectMenu } from '@/components/SearchSelectMenu';
 
+import { isBlockedGeo, signCompliancePayload } from '@/lib/compliance';
+
 type ElementProps = {
   setIsOpen?: (open: boolean) => void;
 };
 
-const CountrySelector = ({ label }: { label: string }) => {
+const CountrySelector = ({
+  label,
+  selected,
+  onSelect,
+}: {
+  label: string;
+  selected: string;
```

Comment:
[suggestion] selectedCountry

the word `selected` can be interpreted as a bool